app/components/dsfr_component/proconnect_button_component.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= content_tag(:div, class: "fr-connect-group", **html_attributes) do %>
+  <%= button_to(target, button_attributes) do %>
+    <span class="fr-connect__login">S'identifier avec</span>
+    <span class="fr-connect__brand">ProConnect</span>
+  <% end %>
+
+  <p>
+    <%= link_to "Qu'est-ce que ProConnect ?", "https://www.proconnect.gouv.fr/", target: "_blank", rel: "noopener", title: "Qu’est-ce que ProConnect ? - nouvelle fenêtre" %>
+  </p>
+<% end %>
+
+<style>
+  .fr-connect {
+  padding-left: 4.375em !important;
+  }
+
+  .fr-connect:before {
```

Comment:
Est-ce qu’on n'utiliserait pas une autre classe pour ça, afin d’éviter de rentrer en conflit avec un éventuel bouton ProConnect se trouvant sur la même page ?

Côté RDV SP nous avons fait ça : 
- https://github.com/betagouv/rdv-service-public/blob/production/app/views/common/_proconnect_button_dsfr.html.slim
- https://github.com/betagouv/rdv-service-public/blob/production/app/javascript/stylesheets/vendor/proconnect_button_dsfr.scss

Reply:
OK merci, pile le genre de retours "in the wild" que j'attendais, je m'en occupe